src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -926,6 +926,14 @@ public boolean isFinished() {
     }
 
     public boolean repeatUntilResponseIs(final String expectedResponse) {
+        return repeatUntil(getRepeatCompletion(expectedResponse));
+    }
+
+    public boolean repeatUntilResponseIsNot(final String expectedResponse) {
+        return repeatUntilNot(getRepeatCompletion(expectedResponse));
+    }
+
+    private RepeatCompletion getRepeatCompletion(final String expectedResponse){
```

Comment:
Very close, only the name could be more descriptive and in general I try to make all methods protected so people can override if desired.

```suggestion
    protected RepeatCompletion createResponseIsCompletion(final String expectedResponse) {
```